ui/app/templates/components/clients/current.hbs

```diff
@@ -49,7 +49,7 @@
               class="column"
               @label="Unique entities"
               {{! ARG TODO naming is changing }}
-              @value={{or @model.activity.distinct_entities "0"}}
+              @value={{or @model.monthly.distinctEntities "0"}}
```

Comment:
I have in my pr that the end part is the same. e.g. montly.distinct_entities. Is this intentional that you camelCased them?

Reply:
Ember does that cool thing where it on the response it's snake cased, but then it maps to the model camel cased - so I was just keeping conventional syntax and use camel case in hbs and js files